test/functionalTest/cases/0000_ngsi10_subscriptions/subscription_sequence_wildcards_onchange_append.test

```diff
@@ -30,87 +30,102 @@ accumulatorStart
 
```

Comment:
Is there any real change in this .test or just cleaning? (In theory, the cache funcationality shouldn't affect to existing functional tests).